postgres/create_tables.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS users (
+	userID INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
how does this exactly index the users? are we taking the discord-indexed ids or will the users just index themselves?